docs/user-manuals/slo-config.md

```diff
@@ -0,0 +1,408 @@
+# SLO Configuration
+
+## Introduction
+
+Koordinator uses a ConfigMap to manage the SLO configurations. The ConfigMap is stored in`${ConfigNamespace}/${SLOCtrlConfigMap}`
+(`koordinator-system/slo-controller-config` by default), with the following keys respectively:
+
```

Comment:
configmap is for slo-controller, which can specify ns/name via startup args

Reply:
fixed